[2025-03-07T15:53:20.599+0000] {processor.py:161} INFO - Started process (PID=165) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:20.600+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:53:20.603+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:53:20.602+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:22.905+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:53:22.904+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 22, in <module>
    AWS_ACCESS_KEY = Variable.get("s3_access_key_team")
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable s3_access_key_team does not exist'
[2025-03-07T15:53:22.905+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:22.914+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 2.329 seconds
[2025-03-07T15:53:53.807+0000] {processor.py:161} INFO - Started process (PID=183) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:53.808+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:53:53.810+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:53:53.809+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:54.381+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:53:54.380+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 22, in <module>
    AWS_ACCESS_KEY = Variable.get("s3_access_key_team")
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable s3_access_key_team does not exist'
[2025-03-07T15:53:54.382+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:53:54.390+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.591 seconds
[2025-03-07T15:54:25.112+0000] {processor.py:161} INFO - Started process (PID=199) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:25.114+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:54:25.116+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:54:25.116+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:25.718+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:54:25.717+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 23, in <module>
    AWS_SECRET_KEY = Variable.get("s3_secret_key_team")
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable s3_secret_key_team does not exist'
[2025-03-07T15:54:25.718+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:25.730+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.625 seconds
[2025-03-07T15:54:56.390+0000] {processor.py:161} INFO - Started process (PID=215) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:56.392+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:54:56.393+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:54:56.393+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:57.018+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:54:57.017+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 26, in <module>
    API_KEYS = Variable.get('api_4_account_key_mix', deserialize_json=True)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable api_4_account_key_mix does not exist'
[2025-03-07T15:54:57.018+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:54:57.027+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.644 seconds
[2025-03-07T15:55:27.734+0000] {processor.py:161} INFO - Started process (PID=231) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:27.736+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:55:27.737+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:55:27.737+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:28.372+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:55:28.371+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 26, in <module>
    API_KEYS = Variable.get('api_4_account_key_mix', deserialize_json=True)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable api_4_account_key_mix does not exist'
[2025-03-07T15:55:28.373+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:28.382+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.656 seconds
[2025-03-07T15:55:59.013+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:59.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:55:59.016+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:55:59.015+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:59.550+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:55:59.549+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/collect_match_info.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/collect_match_info.py", line 26, in <module>
    API_KEYS = Variable.get('api_4_account_key_mix', deserialize_json=True)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable api_4_account_key_mix does not exist'
[2025-03-07T15:55:59.550+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:55:59.558+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.553 seconds
[2025-03-07T15:56:30.306+0000] {processor.py:161} INFO - Started process (PID=263) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:56:30.307+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:56:30.309+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:30.308+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:56:31.007+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:56:31.139+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.138+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:fetch_nexon_data_sequential
[2025-03-07T15:56:31.162+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.162+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:fetch_nexon_data_sequential
[2025-03-07T15:56:31.172+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.171+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:fetch_nexon_data_sequential
[2025-03-07T15:56:31.172+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.172+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:56:31.186+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.186+0000] {dag.py:3118} INFO - Creating ORM DAG for fetch_nexon_data_sequential
[2025-03-07T15:56:31.199+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:56:31.199+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:56:31.221+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.924 seconds
[2025-03-07T15:57:01.486+0000] {processor.py:161} INFO - Started process (PID=279) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:01.487+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:57:01.489+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:01.488+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:02.117+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:02.129+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:02.128+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:57:02.153+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:02.153+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:57:02.173+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.697 seconds
[2025-03-07T15:57:32.749+0000] {processor.py:161} INFO - Started process (PID=295) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:32.752+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:57:32.754+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:32.753+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:33.351+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:57:33.361+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:33.360+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:57:33.381+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:57:33.381+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:57:33.398+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.657 seconds
[2025-03-07T15:58:04.026+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:04.027+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:58:04.029+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:04.028+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:04.715+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:04.727+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:04.726+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:58:04.751+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:04.750+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:58:04.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.776 seconds
[2025-03-07T15:58:35.283+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:35.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:58:35.287+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:35.287+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:35.875+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:58:35.887+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:35.885+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:58:35.907+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:58:35.907+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:58:35.924+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.654 seconds
[2025-03-07T15:59:06.649+0000] {processor.py:161} INFO - Started process (PID=348) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:06.650+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:59:06.652+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:06.651+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:07.283+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:07.294+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:07.294+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:59:07.342+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:07.342+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:59:07.397+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.757 seconds
[2025-03-07T15:59:38.154+0000] {processor.py:161} INFO - Started process (PID=364) to work on /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:38.156+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/collect_match_info.py for tasks to queue
[2025-03-07T15:59:38.157+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:38.157+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:38.739+0000] {processor.py:840} INFO - DAG(s) 'fetch_nexon_data_sequential' retrieved from /opt/airflow/dags/collect_match_info.py
[2025-03-07T15:59:38.750+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:38.750+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-03-07T15:59:38.773+0000] {logging_mixin.py:188} INFO - [2025-03-07T15:59:38.773+0000] {dag.py:3954} INFO - Setting next_dagrun for fetch_nexon_data_sequential to 2025-03-06 00:00:00+00:00, run_after=2025-03-07 00:00:00+00:00
[2025-03-07T15:59:38.790+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/collect_match_info.py took 0.648 seconds
